UNRELEASED.md

```diff
@@ -2,12 +2,17 @@
 
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-# Breaking
+# Breaking:
 
-- Bump `stylelint-config-standard` from `21.0.0` to `22.0.0`
-  - This breaking change brings an upgrade to `stylelint-config-recommended@5.0.0` which includes and enforces the following two new rules `no-invalid-position-at-import-rule` and `no-irregular-whitespace` 
+The major change here is updating Stylelint to `v14`, resulting in a major version update of this package from `v4` to `v5`. A [migration guide](./docs/migration-from-4-to-5.md) has been added.
```

Comment:
Can we update here that the other major change here is moving to scope? This is a nice TLDR; for folks before being able to dive into the migration and do the work

Reply:
⚡